lib/synapse_pay_rest/http_client.rb

```diff
@@ -7,10 +8,11 @@ class HTTPClient
     attr_accessor :options
     attr_accessor :headers
     attr_accessor :user_id
+    attr_accessor :timeout
 
     def initialize(options, base_url, user_id: nil)
       @options = options
-      user = '|%s' %options['fingerprint']
+      user = '|%s' % options['fingerprint']
```

Comment:
Does this equal to "|#{options['fingerprint']}", if so can we do this instead since  `%` method is rarely in use and probably confusing to a lot of people